lib/cronofy/client.rb

```diff
@@ -1093,6 +1093,23 @@ def real_time_scheduling(args = {})
       parse_json(AddToCalendarResponse, nil , response)
     end
 
+    # Public: Creates a embed_token to allow embedding of account related UI components
+    #
+    # See https://www.cronofy.com/developers/api/alpha/#embed-tokens for
+    # reference.
+    #
+    # Returns an embed token
+    #
+    # Raises Cronofy::CredentialsMissingError if no credentials available.
+    # Raises Cronofy::AuthenticationFailureError if the access token is no
+    # longer valid.
+    # Raises Cronofy::TooManyRequestsError if the request exceeds the rate
+    # limits for the application.
+    def embed_token
```

Comment:
These have never been released externally and won't be as UI Element tokens are the preferred path so support should be removed.

Reply:
Removed